tracee-rules/Makefile

```diff
@@ -8,11 +8,11 @@ OUT_DIR ?= dist
 OUT_BIN := $(OUT_DIR)/tracee-rules
 OUT_RULES := $(OUT_DIR)/rules
 GO_SRC := $(shell find . -type f -name '*.go')
-GOSIGNATURES_DIR ?= signatures/golang
+GOSIGNATURES_DIR ?= ../signatures/golang
 GOSIGNATURES_SRC := $(shell find $(GOSIGNATURES_DIR) -type f -name '*.go' ! -name '*_test.go' ! -path '$(GOSIGNATURES_DIR)/examples/*')
 OUT_GOSIGNATURES := $(OUT_RULES)/builtin.so
-REGO_SIGNATURES_DIR ?= signatures/rego
-REGO_SIGNATURES_SRC := $(shell find $(REGO_SIGNATURES_DIR) -type f -name '*.rego' ! -name '*_test.rego' ! -path '$(REGO_SIGNATURES_DIR)/examples/*')
+REGO_SIGNATURES_DIR ?= ../signatures/rego
```

Comment:
I think we must also adjust the working directory for the `opa test` command. It seems that we no longer run any tests for Rego.

Instead of

```
opa test . --verbose --ignore="examples" --ignore="dist" --ignore="benchmark"
```

use something like?

```
opa test ./../signatures --verbose --ignore="examples" --ignore="dist" --ignore="benchmark"
```